tagcompare/main.py

```diff
@@ -1,10 +1,115 @@
+import argparse
+import sys
+import logging
+
 import capture
 import compare
+import settings
+import logger
+
+
+LOGGER = logger.Logger(name="main", writefile=True).get()
+
+
+def __query_yes_no(question, default="yes"):
```

Comment:
I looked, and this is what the internet came up with.  I took this code from (also noted in the method documentation): http://code.activestate.com/recipes/577058/